### PR TITLE
MI300X custom pbqt conf.

### PR DESCRIPTION
**Pull request by jkottiku**
pebb & pbqt result log indentation.

### CODE REVIEW

File: pbqt.so/src/action.cpp [hunk 1]
```diff
@@ -508,9 +512,13 @@ int pbqt_action::create_threads() {
       // perform peer check
       if (is_peer(gpu_id[i], gpu_id[j])) {
         RVSTRACE_
+
+        snprintf(srcgpuid_buff, sizeof(srcgpuid_buff), "%5d", gpu_id[i]);
```
Comment (manoj-freyr) [inline pbqt.so/src/action.cpp:516]: see if you can use std::setw() instead of %5d ?

File: pbqt.so/src/action.cpp [hunk 2]
```diff
@@ -881,10 +897,14 @@ int pbqt_action::print_final_average() {
     transfer_ix = (*it)->get_transfer_ix();
     transfer_num = (*it)->get_transfer_num();
 
+    snprintf(transfer_buff, sizeof(transfer_buff), "%2d", transfer_ix);
```
Comment (manoj-freyr) [inline pbqt.so/src/action.cpp:900]: can we use setw() when printing instead of %d qualifiers?

**Review by manoj-freyr — CHANGES_REQUESTED** [review]

File: pbqt.so/src/action.cpp (hunk 1 above)
Comment (jkottiku) [reply pbqt.so/src/action.cpp:516]: Yes, I tried it. But we need addition steps made the code kind of complex. This implementation easily fit into the code (especially **_msg_** log string) we have in place.

**Review by manoj-freyr — APPROVED** [review]
looks good

**Review by manoj-freyr — APPROVED** [review]

**Pull request merged**